server/graphql/v1/mutations/expenses.js

```diff
@@ -487,6 +479,17 @@ export async function deleteExpense(req, expenseId) {
 async function markExpenseAsPaid(expense, remoteUser) {
   debug('update expense status to PAID', expense.id);
   await expense.setPaid(remoteUser.id);
+
+  // Add user as CONTRIBUTOR
+  const contributor = await models.User.findOne({ where: { CollectiveId: expense.FromCollectiveId } });
```

Comment:
We now support creating expenses as organizations, which means `FromCollectiveId` is sometimes referring to a profile that is not directly linked to a user.

I'd recommend using `expense.UserId` (you can fetch it with `expense.getUser()`).